search/node-search.go

```diff
@@ -26,8 +28,31 @@ type keyValue struct {
 	value *yaml.Node
 }
 
+type Seeking int
+
+// Indicates to search what component of key-value pairing findcomponent should return
```

Comment:
match casing for findComponent and is 'search' still accurate in this comment

Reply:
got rid of seeking